### ISSUE TITLE
Gef can't import ROPgadget.

### ISSUE THREAD

**Issue by DarkLinkXXXX** [hugsy/gef#2]
```
┌[kiwish-4.2]-(Code/gef)-[git:master]-$
└> pip2 install ROPgadget
Requirement already satisfied (use --upgrade to upgrade): ROPgadget in /usr/local/lib/python2.7/dist-packages
Requirement already satisfied (use --upgrade to upgrade): capstone in /usr/local/lib/python2.7/dist-packages (from ROPgadget)
Cleaning up...
┌[kiwish-4.2]-(Code/gef)-[git:master]-$
└> gdb -q /bin/ls
[+] Failed to load `ropgadget`: 'Failed to import ROPgadget (check path)'
gef loaded, `gef help' to start, `gef config' to configure
28 commands loaded (10 sub-commands), using Python engine 2.7
Reading symbols from /bin/ls...(no debugging symbols found)...done.
gef> quit
┌[kiwish-4.2]-(Code/gef)-[git:master]-$
└> python
Python 2.7.9 (default, Mar  1 2015, 12:57:24) 
[GCC 4.9.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import ropgadget
>>> 
┌[kiwish-4.2]-(Code/gef)-[git:master]-$
└> 
```

**Comment by hugsy** [comment]
Fixed by 5dfdef5. I added a `help` subcommand to see all the options supported.
```
gef> ropgadget help
```

Thanks for reporting.

**Comment by DarkLinkXXXX** [comment]
That fixed it. Thank you very much! :beer:
